tests/test_edgeql_json.py

```diff
@@ -832,6 +832,19 @@ async def test_edgeql_json_object_unpack_05(self):
             [1, 2]
         )
 
+    async def test_edgeql_json_object_pack(self):
```

Comment:
We'll want some more tests. Try one with empty set input, with input from an `array_unpack`, and from an object property. (Object property is good because it will give you something that is definitely packed up as values, not as a TupleVar)